### ISSUE TITLE
Spaces is slice operator

### ISSUE THREAD

**Issue by kotofos** [google/yapf#172]
Is'n this code from readme contradict pip8 about spaces around slice operator?
def f(a):
    return 37 + -+a[42 - x:y ** 3]

exactly:
https://www.python.org/dev/peps/pep-0008/#pet-peeves item 3:

No:
ham[lower + offset:upper + offset]

**Comment by gwelymernans** [comment]
Yeah. I noticed this before. I super disagree with it, but it's not my PEP document. :-) There are a few formatting decisions that conflict here. But I think we can implement something that may help here...

**Comment by janhavi-naik14** [comment]
Spaces in the context of slicing can lead to confusion or readability issues but typically do not cause functional errors in Python. The key is to ensure that colons are used correctly and consistently in slice operations to avoid any syntax errors or misunderstandings.
